test/utils/ValidatorTester.js

```diff
@@ -332,6 +332,26 @@ const ValidatorTester = (requestEndpoint) => (location) => (field_name) => ({
             });
         });
     },
+
+    mustBeGreaterThanOrEqualTo: (min) => {
```

Comment:
Don't we have any case where we test the max one?

Reply:
I don't think so. We don't have any case for that one, so I don't think we should add the code. If necessary, It's very easy to create it by looking at this one